### PR TITLE
Technical Staff Position for QuTiP available at RIKEN

### PR DESCRIPTION
**Pull request by nathanshammah**
- Add news on a new Technical Staff Position for QuTiP available at RIKEN, Japan. 
- Add job description in the jobs page
- Added navigator header to homepage 
- Added the job post also on Quantiki and included the link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +37,16 @@
     document.getElementById("cite").innerHTML = txt;
 </script>
 
+
+<script>
+    var lng = navigator.language;
+    var txt;
+    {
+        txt = "A Technical Staff for QuTiP is available, check it out under the <a href='jobs.html'>jobs</a> page.";
```
Comment (hodgestar) [inline index.html:45]:
```suggestion
        txt = "A Technical Staff position for QuTiP is available, check it out under the <a href='jobs.html'>jobs</a> page.";
```

File: index.html [hunk 2]
```diff
@@ -34,6 +37,16 @@
     document.getElementById("cite").innerHTML = txt;
 </script>
 
+
+<script>
+    var lng = navigator.language;
```
Comment (hodgestar) [inline index.html:42]:
```suggestion
```
It doesn't look like `lng` is used for anything here?

File: jobs.html [hunk 3]
```diff
@@ -1,12 +1,46 @@
...
 <h1 style="margin-bottom:20px">Job Announcements</h1>
+
+<h4>Technical Staff for QuTiP</h4>
+
+
+<p><strong>About this job</strong></p>
+<p>This is a job post for a [one]-year job opportunity to work on maintaining and improving QuTiP (the Quantum Toolbox in Python). Compensation is competitive with academic community standards and will depend on the candidate experience (PhD graduate or not, etc.). The candidate will work at RIKEN, Wako campus, Japan, within the Theoretical Quantum Physics Laboratory, RIKEN Cluster for Pioneering Research (Laboratory head: Franco Nori, Chief Scientist). RIKEN is Japan&rsquo;s basic science research institute. Wako is located in the Tokyo suburban area. You will coordinate with the QuTiP admin developers, under the supervision of Prof. Franco Nori.</p>
+<p><strong>Responsibilities</strong></p>
+<p>Coordinate with the QuTiP admin developers, under the supervision of Prof. Franco Nori, to maintain and improve QuTiP&rsquo;s features and functionalities, such as the ones already prioritized at <a href="https://github.com/qutip/qutip/wiki/Google-Summer-of-Code-2021">https://github.com/qutip/qutip/wiki/Google-Summer-of-Code-2021</a>.</p>
```
Comment (hodgestar) [inline jobs.html:17]: @ajgpitch and I discussed moving the GSoC 2021 project list into qutip-doc. No need to change this now, just reminding us that we might need to update the reference at some point.

File: jobs.html [hunk 4]
```diff
@@ -1,12 +1,46 @@
...
+<li> Code development experience working on a large collaborative open-source project</li>
+<li> Track record of involvement or support of the QuTiP community (Github code contributions, Q&amp;As on QuTiP&rsquo;s forum, etc.)</li>
+<li> Experience with developing and testing numerically efficient code such as: Cython, multithreading with OpenMP, PyPy.</li>
+</ul>
+<p><strong>Contact</strong></p>
+<p>Applicants should contact fnori@riken.jp directly with a CV and cover letter. Related job opportunities (postdoc only) can be found at: <a href="https://www.riken.jp/en/careers/researchers/20200914_2/index.html">https://www.riken.jp/en/careers/researchers/20200914_2/index.html</a> for a postdoc working on coherent Ising machines, and neural networks and machine learning applied to quantum physics. Candidates with prior experience in these topics will have priority.</p>
+
+
+
+This job post can also be found on <a href='https://www.quantiki.org/position/technical-staff-qutip'> Quantiki</a>.
```
Comment (hodgestar) [inline jobs.html:39]:
```suggestion
This job post can also be found on <a href='https://www.quantiki.org/position/technical-staff-qutip'>Quantiki</a>.
```

File: news.html [hunk 5]
```diff
@@ -11,6 +11,18 @@ <h1 style="margin-bottom:20px">QuTiP News</h1>
 </div>
 <br>
 
+<div class="row" style='margin-bottom: 15px'>
+    <div class="col-md-8">
+        <p style='margin-bottom: -10px'>March 18, 2021</p>
+        <h4>Technical Staff Position for QuTiP Available at RIKEN
+</h4>
```
Comment (hodgestar) [inline news.html:18]: I would move the closing `</h4>` onto the line above.

**Review by hodgestar — CHANGES_REQUESTED** [review]
Looks good! Is this a colleague for me? That would be awesome.

File: news.html (hunk 5 above)
Comment (nathanshammah) [inline news.html:18]:
```suggestion
        <h4>Technical Staff Position for QuTiP Available at RIKEN</h4>
```

**Comment by nathanshammah** [comment]
Thanks @hodgestar. 
> Looks good! Is this a colleague for me? That would be awesome.
Yes, starting later on I think.

**Review by hodgestar — APPROVED** [review]

**Comment by nathanshammah** [comment]
Ouch, the navigator banner does not show up nice on the homepage, http://qutip.org/. @hodgestar if you have any idea on how to fix that, you're more then welcome to edit it.

**Comment by hodgestar** [comment]
> Ouch, the navigator banner does not show up nice on the homepage, http://qutip.org/. @hodgestar if you have any idea on how to fix that, you're more then welcome to edit it.

I've opened #137 (~ 1 over fine structure constant :) with a fix.

**Pull request merged**